test/FBinop.lean

```diff
@@ -52,6 +52,22 @@ example (s : Finset α) (t : Finset (α × ℕ)) : s ×ˢ' {1, 2, 3} = t := test
 example (s : Finset α) (t : Finset (ℕ × α)) : {1, 2, 3} ×ˢ' s = t := test_sorry
 example (s : Finset α) (_t : Finset (ℕ × α)) : ({1, 2, 3} ×ˢ' s).card = 22 := test_sorry
 
+-- FIXME nightly-testing
+-- The following app_unexpanders have been lost in the shuffle.
+-- Copy pasted here to mitigate the test, and Scott is following up.
```

Comment:
```suggestion
-- FIXME nightly-testing
-- The following app_unexpanders have been lost in the shuffle.
-- Copy pasted here to mitigate the test.
-- This will be fixed on nightly-2024-02-28
```